app/javascript/articles/Article.jsx

```diff
@@ -49,10 +50,14 @@ export const Article = ({
         role="presentation"
         onClick={(event) => {
           const { classList } = event.target;
-
           if (clickableClassList.includes(...classList)) {
-            InstantClick.preload(article.path);
-            InstantClick.display(article.path);
+            if (event.which > 1 || event.metaKey || event.ctrlKey) { // Indicates should open in _blank
```

Comment:
`event.which > 1` is "middle click" on mouse. `event.metaKey` and `event.ctrlKey` account for command/ctrl + click.